Team8745/src/main/java/org/firstinspires/ftc/team8745/TeleOp_HDrive.java

```diff
@@ -0,0 +1,93 @@
+package org.firstinspires.ftc.team8745;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+/**
+ * Created by rose on 10/7/17.
+ */
+@TeleOp(name = "H-Drive Test Oct. 7")
+public class TeleOp_HDrive extends OpMode {
+
+    private Robot robot = new Robot();
+
+    @Override
+    public void init() {
+        robot.init(hardwareMap);
+
+
+    }
+    //Comment line below.
+    boolean wasBumper = false;
+    boolean sideMode = false;
+
+    // If one value is greater than another and the first value is greater than the third value, return 1. Otherwise, 0.
+    public static int test(float stickL, float stick1, float stick2) {
+        int yes = 0;
+        if (Math.abs(stickL)!=0&&Math.abs(stickL)>Math.abs(stick1)&&Math.abs(stickL)>Math.abs(stick2)){
+            yes = 1;
+        }
+        else {
+            yes = 0;
+        }
+        return yes;
+    }
+    @Override
+    public void loop() {
+        float r_stick = gamepad1.right_stick_y;
```

Comment:
Use camelCase instead of underscore_notation please... In this case you would use rightStick instead of r_stick.